src/transformers/modeling_xlm.py

```diff
@@ -512,7 +512,7 @@ def forward(
             inputs_embeds = self.embeddings(input_ids)
 
         tensor = inputs_embeds + self.position_embeddings(position_ids).expand_as(inputs_embeds)
-        if langs is not None and self.use_lang_emb:
+        if langs is not None and self.use_lang_emb and self.n_langs > 1:
```

Comment:
I had to include this if statement to not get a `self.lang_embeddings` is undefined error. Is this fine @LysandreJik ?

Reply:
I believe this is the exact same I did for [FlauBERT](https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_flaubert.py#L234), too bad I missed XLM!